### PR TITLE
Create recipes to test more features of the vsphere driver

### PR DESCRIPTION
**Pull request by josh-barker**
I've created some more test cases for the vsphere driver.

However, we'll need to 
- Create a customization spec in vCenter & update the recipe
- Upload an ISO (perhaps the centos iso) & update the recipe
- Update `centos_provision_with_customization_spec` with proper IP address values

### CODE REVIEW

**Comment by josh-barker** [comment]
I've also got an open PR that we can use for testing - https://github.com/chef-partners/chef-provisioning-vsphere/pull/88

**Comment by jjasghar** [comment]
Awesome, i'll circle back on this ASAP. Thanks!

File: cookbooks/provision/recipes/centos_provision_with_disks.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                               template_name: 'centos7-tmpl',
+                               :ssh => {
+                                 :user => 'root',
+                                 :password => 'admini',
+                                 :paranoid => false,
+                               },
+                               bootstrap_ipv4: true,
+
+                               additional_disk_size_gb: [10, 20],
+                             }
```
Comment (jjasghar) [inline cookbooks/provision/recipes/centos_provision_with_disks.rb:29]: added the line:
`                               datastore: 'vsanDatastore',` here. 

I got it to pass but didn't see the disk's added. Could be broken code though.

**Comment by jjasghar** [comment]
Closing this in favor for https://github.com/jjasghar/tirefire-provision-cookbook/pull/2

**Pull request closed without merging**